### PR TITLE
Do a kubectl get nodes after turning up the e2e cluster.

### PR DESCRIPTION
**Pull request by spxtr**
https://github.com/kubernetes/test-infra/issues/366

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/34128)

### CODE REVIEW

**Comment by spxtr** [comment]
https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/34128/kubernetes-pull-build-test-e2e-gce/61882/

**Comment by zmerlynn** [comment]
Ha, I wish I had seen this before my comment in #34153. This is the beginning of the solution to the cluster-down issue in that PR, though something will need to pull the variables out of the artifacts `.yaml` instead.

File: hack/e2e.go [hunk 1]
```diff
@@ -211,6 +211,20 @@ func run(deploy deployer) error {
 		if err := xmlWrap("Up", deploy.Up); err != nil {
 			return fmt.Errorf("starting e2e cluster: %s", err)
 		}
+		if *dump != "" {
```
Comment (zmerlynn) [inline hack/e2e.go]: Do we maybe want to pull this out to one level higher and run it in either the case that `-up` **or** `-test` is selected, then just eliminate the `kubectl version` check for `-test`? i.e. move it to about line 215 in the original and just say:
```
if *up || *test {
  // get nodes. If there's something wrong, we need to get out of dodge after this.
  // if *dump is set, send the output to a file
}
```

Comment (zmerlynn) [inline hack/e2e.go]: Actually, that may not be the best for large clusters, where `get nodes -oyaml` might be dicey in the first place. Hrm.

Comment (spxtr) [inline hack/e2e.go]: Ah, I didn't consider large clusters. How bad is it?

**Comment by spxtr** [comment]
@k8s-bot test this

File: hack/e2e.go (hunk 1 above)
Comment (spxtr) [inline hack/e2e.go]: OK. No worries for the delay. I'm not in a rush.

**Pull request merged**